---
apiVersion: "v1"
kind: "Event"
metadata:
  labels:
    EntandoResourceKind: "TestResource"
    EntandoResourceNamespace: "my-namespace"
    TestResource: "my-app"
  name: "my-app-04235583"
  namespace: "my-namespace"
  ownerReferences:
  - apiVersion: "test.org/v1"
    kind: "TestResource"
    blockOwnerDeletion: true
    controller: true
    name: "my-app"
    uid: "3a2c5543-87cc-44f8-9114-fb21bd376e97"
action: "FAILED"
count: 1
firstTimestamp: "2021-07-01T10:19:01Z"
involvedObject:
  apiVersion: "test.org/v1"
  kind: "TestResource"
  fieldPath: "status"
  name: "my-app"
  namespace: "my-namespace"
  resourceVersion: "4289194"
  uid: "3a2c5543-87cc-44f8-9114-fb21bd376e97"
lastTimestamp: "2021-07-01T10:19:01Z"
message: "The deployment of TestResource my-namespace/my-app failed due to Deployment\
  \ failed. Please inspect the logs of the pod my-namespace/my-app-deployment. Fix\
  \ the root cause and then trigger a redeployment by adding the annotation 'entando.org/processing-instruction:\
  \ force'"
reason: "Failed"
related:
  apiVersion: "v1"
  kind: "Pod"
  name: "test-controller-pod"
  namespace: "controller-namespace"
source:
  component: "entando-k8s-ource-controller"
type: "Error"
